src/libsystemd/sd-journal/journal-file.c

```diff
@@ -2078,6 +2078,19 @@ static int entry_item_cmp(const EntryItem *a, const EntryItem *b) {
         return CMP(le64toh(a->object_offset), le64toh(b->object_offset));
 }
 
+static size_t remove_duplicate_entry_items(EntryItem *items, size_t n) {
```

Comment:
doesn't matter, but if you'd replace `*items` by `items[]` here, things would be a tiny bit clearer, even though of course it's entirely equivalent